src/input/datetime.rs

```diff
@@ -1,14 +1,13 @@
+use super::Input;
+use crate::errors::{ErrorType, ValError, ValResult};
```

Comment:
please move these down to where they were before, they match the order of other imports.